README.md

```diff
@@ -26,3 +26,49 @@ Launch Node-RED with the `--settings|-s` command line argument:
 
 [1]:https://nodered.org/#get-started
 [2]:https://nodered.org/docs/configuration
+
+
+
+# Connect DS18b20 to RPI and node-red.
+
+## CONFIGURATION device in GPIO
+
+### DS18b20 wire circuit :
+
+	DS18b20 / breadboard (column(from right),row(from bottom) or a10,b22,etc)
+	gnd last, 4th 
+	vcc first, 3rd
+	sig a4
+
+![ds18b20](img/wire_circuit/ds18b20.jpg)
+![overview](img/wire_circuit/overview.jpg)
+![Breadborad pin_a4](img/wire_circuit/pin_a4.jpg)
+
+### Configuration device in RPI
+	read the following document from page 11 (pagg. 1-10 not compatible DS18b20 wire circuit) :
+
+	doc/adafruits-raspberry-pi-lesson-11-ds18b20-temperature-sensing.pdf
+
+	configuration device on RPI: pag. 11-12
+	script Python : pag. 13-15
+
+### Configuration device in node-red
+	1)run a bash shell
```

Comment:
Suggested reading https://github.com/google/styleguide/blob/gh-pages/docguide/style.md

Reply:
to improve the markdown of this PR, of course :)